release-notes/2024-09/README.md

```diff
@@ -0,0 +1,221 @@
+- Use the following new features when [visualizing data tables in data spaces](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/visualizing-data-tables-in-a-data-space.html).
+    - Search for columns to plot from the axis selection grid.
+    - Color traces by data table and by column metadata.
+- Adjust how long the SystemLink Enterprise Alarm Service keeps your alarms - [Configuring Alarm Retention](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/configuring-alarm-retention.html).
+    - [View this service configuration](https://github.com/ni/install-systemlink-enterprise/blob/2024-09/getting-started/templates/systemlink-values.yaml#L469)
+- Adjust how many alarms you can create in the SystemLink Enterprise Alarm Service - [Configuring Alarm Limits](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/configuring-alarm-limits.html).
+    - [View this service configuration](https://github.com/ni/install-systemlink-enterprise/blob/2024-09/getting-started/templates/systemlink-values.yaml#L484)
+- Learn about the [performance metrics for the Dashboard Host Alarm Service](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/performance-metrics-dashboard-host-service.html).
+- Privilege **List published notebooks in SystemLink and download published notebooks into the Jupyter development environment** was split to **List available notebooks** to list the published notebooks, and **List published notebooks in SystemLink and download published notebooks into the Jupyter development environment** to list and download notebooks for notebook development.
+    - This privilege is enabled by default for Collaborator, Data Maintainer, and Systems Maintainer built-in roles.
+    - For the Automated Agent, the **List published notebooks in SystemLink and download published notebooks into the Jupyter development environment** privilege will be removed, as the Automated Agent should not need to download the notebooks content.
+- Support for [Private Certificate Authorities](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/private-certificate-authorities.html).
+    - Connect to MongoDB and S3 compatible interfaces that use certificates signed by a private authority.
+    - Call SystemLink APIs from Jupyter Notebook when the SystemLink API endpoint is configured to use a certificate signed by a private authority.
+    - Trusted certificates specified at `global.trustedCertificatesSecrets` are now installed on servers that use Jupyter. These certificates are used when calling SystemLink or other external APIs from notebooks.
+    - Trusted certificates specified at `global.trustedCertificatesSecrets` are now installed on execution pods. These certificates are used when calling SystemLink or other external APIs from the pods.
+- Added abilities to configure the rate-limit values for notebook APIs inside the Helm chart.
```

Comment:
We need a reference to where and how this is done. This likely will require us to update the system link-value.yaml in this repo to expose this configuration. Alternatively we could reference the ni.com doc on this configuration should one exist.

Reply:
I did ask Hem Kumar Rao on Teams to check this input - for some reason I cannot tag his user: hemkumarrao-ni

Reply:
This is just a rate-limiting implementation for some APIs(notebook APIs) in the WebApp service. This is very much similar to the rate-limiting done in the Notebook Execution Service mentioned just below in this document. This is not an enterprise helm level change but a service helm level change. 

Reference PR - [Pull Request 787094](https://dev.azure.com/ni/DevCentral/_git/Skyline/pullrequest/787094): WebApp Service | Configure rate limit via Helm chart

We can rephrase it into 
`- WebApp Service
      - Implemented Rate limiting for the notebook APIs`
      
 @prestwick @CIakab-NI @MuthuRajan-S @JD-Robertson

Reply:
Thanks, @hemkumarrao-ni > I did replace the initial line with the text you had suggested:
- Implemented Rate limiting for the Notebook APIs, in the WebApp Service.

Reply:
FYI I'm going to adjust the wording of this line. We also lack docs for this. I am not going to hold the release for that.  Bug https://dev.azure.com/ni/DevCentral/_workitems/edit/2605257 filed.